TCAT/Views/DatePickerView.swift

```diff
@@ -183,13 +183,14 @@ class DatePickerView: UIView {
 
     @objc private func doneButtonPressed() {
         var searchTimeType: SearchType = .leaveNow
-        switch leaveNowSegmentedControl.selectedSegmentIndex {
-        case leaveNowElement.index:
+        if leaveNowSegmentedControl.selectedSegmentIndex == leaveNowElement.index {
             searchTimeType = .leaveNow
-        case arriveByElement.index:
-            searchTimeType = .arriveBy
-        default:
+        } else if timeTypeSegmentedControl.selectedSegmentIndex == leaveAtElement.index {
             searchTimeType = .leaveAt
+        } else if timeTypeSegmentedControl.selectedSegmentIndex == arriveByElement.index {
+            searchTimeType = .arriveBy
+        } else {
+            searchTimeType = .leaveNow
```

Comment:
Can we keep this as a switch rather than a bunch of else ifs?

Reply:
Incorporated an improvement in logic that makes the switch more suitable. I initially had it all as else if's since the first comparison was of a different variable, so thought it'd look better for consistency. Update coming